admin_guide/managing_networking.adoc

```diff
@@ -320,6 +320,14 @@ the number of local MAC addresses that the network interface can handle before
 falling back to filtering MAC addresses in software.
 ====
 
+[IMPORTANT]
+====
+Currently, the egress router is not compatible with Amazon AWS due to AWS not
+compatible with macvlan traffic.
+
+See link:https://aws.amazon.com/answers/networking/controlling-vpc-egress-traffic/[the Amazon AWS documentation for alternate methods for egress routing].
```

Comment:
@danwinship Sure, I get what you mean. Would you suggest wording it that way instead? So:

"See the Amazon AWS documentation for suggested configurations that don't involve egress routing"

Or is there a more precise thing we could suggest?